esrally/track/loader.py

```diff
@@ -1115,6 +1135,24 @@ def meta_data(self):
         }
 
 
+class TrackDependencyRegistry:
+    """
+    Installs track-specific dependencies to the local runtime
+    """
+
+    def __init__(self, install_dependencies):
+        self.install_dependencies = install_dependencies
+
+    def __call__(self, name):
+        if self.install_dependencies:
+            console.info(f"Installing track dependency {name}")
```

Comment:
Is it expected that I'm not seeing this the the CI log? https://elasticsearch-ci.elastic.co/job/elastic+rally+pull-request/24/consoleFull

Reply:
What I know is that this will get muted by `--quiet`, but what I don't know currently is how all the ITs seem to have quiet output, since only a few actually specify it